.kitchen.yml

```diff
@@ -54,19 +51,30 @@ suites:
         server_repl_password: q654321
```

Comment:
(you can't comment on the lines above because they're from the upstream cookbook, not from this PR)